flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/DefaultExecutionGraphConstructionTest.java

```diff
@@ -316,7 +328,8 @@ void testRegisterConsumedPartitionGroupToEdgeManager() throws Exception {
 
         List<JobVertex> ordered = new ArrayList<>(Arrays.asList(v1, v2));
         ExecutionGraph eg = createDefaultExecutionGraph(ordered);
-        eg.attachJobGraph(ordered);
+        eg.attachJobGraph(
+                ordered, UnregisteredMetricGroups.createUnregisteredJobManagerJobMetricGroup());
```

Comment:
consider putting `UnregisteredMetricGroups.createUnregisteredJobManagerJobMetricGroup()` into some static variable and using that.